### PR TITLE
Start stop watch implemented

### PR DESCRIPTION
**Pull request by anthonyrodarte**

### CODE REVIEW

File: stopwatch/main.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+/* eslint-disable no-unused-vars */
+var start = document.querySelector('#start-btn')
+var time = document.querySelector('#timer')
+
+var sec = 0, min = 0, hour = 0
+
+start.addEventListener('click', timer)
+
+function timer () {
+  i = setInterval(count, 1000)
```
Comment (ghost) [inline stopwatch/main.js]: You don't use `i`. Also, if you want to declare a variable you use `var`.

File: stopwatch/main.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+}
+
+function count() {
+  sec++
```
Comment (ghost) [inline stopwatch/main.js]: Don't abbreviate variable names.

File: stopwatch/main.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  if (sec == 60) {
```
Comment (ghost) [inline stopwatch/main.js]: Always use strict equality `===`.

File: stopwatch/main.js [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+var sec = 0, min = 0, hour = 0
+
+start.addEventListener('click', timer)
+
+function timer () {
+  i = setInterval(count, 1000)
+}
+
+function count() {
```
Comment (ghost) [inline stopwatch/main.js]: The function `count` should take some parameters.

File: stopwatch/main.js [hunk 5]
```diff
@@ -0,0 +1,24 @@
+/* eslint-disable no-unused-vars */
+var start = document.querySelector('#start-btn')
+var time = document.querySelector('#timer')
+
+var sec = 0, min = 0, hour = 0
```
Comment (ghost) [inline stopwatch/main.js]: Consider grouping these in an object to make them easier to pass into other functions.

File: stopwatch/main.js [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+  sec++
+  if (sec == 60) {
+    sec = 0
+    min++
+    if (min == 60) {
+      min = 0
+      hour++
+    }
+  }
+  time.textContent = hour + ':' + min + ':' + sec
```
Comment (ghost) [inline stopwatch/main.js]: Return the time string and use it to set the time.textContent in another part of the code.

**Review by ghost — CHANGES_REQUESTED** [review]

**Review by miladmemarian — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Pull request merged**